.github/config/assertoor/network_params_tx.yaml

```diff
@@ -1,12 +1,9 @@
 participants:
   - el_type: geth
-    el_image: ethereum/client-go:v1.15.2
     cl_type: lighthouse
-    cl_image: sigp/lighthouse:v7.0.0-beta.0
     validator_count: 32
   - el_type: ethrex
     cl_type: lighthouse
-    cl_image: sigp/lighthouse:v7.0.0-beta.0
```

Comment:
if you don't pin the versions of the clients, then an upgrade might break the CI. Please update the versions of the clients instead of removing these lines